homeassistant/components/androidtv/media_player.py

```diff
@@ -91,12 +96,25 @@
 DEVICE_FIRETV = "firetv"
 DEVICE_CLASSES = [DEFAULT_DEVICE_CLASS, DEVICE_ANDROIDTV, DEVICE_FIRETV]
 
+DIRECTION_PULL = "pull"
```

Comment:
wouldn't the words `download` and `upload` make more sense ?

Reply:
I split the service into `androidtv.download` and `androidtv.upload` in https://github.com/home-assistant/home-assistant/pull/30086/commits/9dbff3d138c541b294250f956127b27186c086f6